python/export_util.py

```diff
@@ -235,3 +311,287 @@ def csv_query(
+        if element.get("type") == "node":
+            output.write(f'<node id="n{str(element.get("id"))}')
+            if (
+                element.get("labels")
+                and config.get("format").upper() != "TINKERPOP"
+            ):
+                output.write('" labels="')
+                for label in element.get("labels"):
+                    output.write(f":{label}")
+            output.write('">')
+
+            write_labels_as_data(element, output, config)
+
+            for key, value in element.get("properties").items():
+                output.write(
+                    f'<data key="{key}">{get_value_string(value)}</data>'
+                )
+            output.write("</node>\n")
+
+        elif element.get("type") == "relationship":
+            output.write(
+                f'<edge id="e{str(element.get("id"))}" source="n{str(element.get("start"))}" target="n{str(element.get("end"))}" label="{element.get("label")}">'  # noqa: E501
+            )
+
+            if config.get("format").upper() == "TINKERPOP":
+                output.write(
+                    f'<data key="labelE">{element.get("label")}</data>'
+                )
+            else:
+                output.write(
+                    f'<data key="label">{element.get("label")}</data>'
+                )
+            if config.get("format").upper() == "GEPHI":
+                output.write(f'<data key="TYPE">{element.get("label")}</data>')
+
+            for key, value in element.get("properties").items():
+                output.write(
+                    f'<data key="{key}">{get_value_string(value)}</data>'
+                )
+            output.write("</edge>\n")
+
+
+def write_footer(output: io.StringIO):
+    output.write("</graph>\n")
+    output.write("</graphml>")
+
+
+def set_default_config(config: mgp.Map):
+    if not config.get("stream"):
+        config.update({"stream": False})
+    if not config.get("format"):
+        config.update({"format": ""})  # should it be Gephi?
+    if not config.get("caption"):
+        config.update({"caption": []})
+    if not config.get("useTypes"):
+        config.update({"useTypes": False})
+    if not config.get("leaveOutLabels"):
+        config.update({"leaveOutLabels": False})
+    if not config.get("leaveOutProperties"):
+        config.update({"leaveOutProperties": False})
+    if (
+        not isinstance(config.get("stream"), bool)
+        or not isinstance(config.get("format"), str)
+        or not isinstance(config.get("caption"), list)
+        or not isinstance(config.get("useTypes"), bool)
+        or not isinstance(config.get("leaveOutLabels"), bool)
+        or not isinstance(config.get("leaveOutProperties"), bool)
+    ):
+        raise TypeError(
+            "Config parameter must be a map with specific \
+             keys and values described in documentation."
+        )
+
+
+@mgp.read_proc
+def graphml(
+    ctx: mgp.ProcCtx,
+    path: str = "",
+    config: mgp.Map = {},
```

Comment:
See above comment about default dict parameters
```suggestion
    config: mgp.Map = dict(),
```